src/components/Section.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function Section({ image, title, paragraphs }) {
+    if (image) {
+        return (
+            <section>
+                <img src={image} alt="Fills some blank space" />
+            </section>
+        );
+    } else {
+        return (
+            <section>
+                <h2>{title}</h2>
+                {paragraphs.map((paragraph) => {
```

Comment:
Kreeg dit niet voor elkaar. Fijn dat ik het even kan copiëren :-)
Ik kreeg dan ook dezelfde melding als jij, dat React een unieke key nodig heeft.
Het duurde even, maar mijn oplossing is om 'let i=0; toe te voegen en dan in de p-tag '<p key={i++}> toe te voegen.
`function Tile( {image, title, paragraphs} ) {
  if ( !image ) {
    let i = 0;
    return (
      <section>
        <h2>{ title }</h2>
        { paragraphs.map( ( paragraph ) => {
          return <p key={i++}>{ paragraph }</p>;
        } ) }
      </section>);`

Gek is wel dat de key niet getoond wordt, maar de error-melding is weg

Reply:
oh wat vreemd? Bij mij gaat het gewoon goed als ik het draai 😕